### ISSUE TITLE
Extending the schema to support a "security-reports" field

### ISSUE THREAD

**Issue by h4ckninja** [composer/composer#6319]
Today, the schema supports a `support` array (https://getcomposer.org/doc/04-schema.md#support). This is great. It has a field for `issues`, `docs`, and more. However, some authors may not appreciate having security reports in their issue tracker (commonly GitHub). The schema doesn't currently allow this distinction.

My proposal is to add a field for `security-reports` or `security` to allow an email address or website where security issues can be reported if the maintainer does not want sensitive issues included with bug reports and feature requests.

**Comment by alcohol** [comment]
I think a readme could more suitably explain what constitutes a security issue, and where and how it should be reported.

**Comment by h4ckninja** [comment]
I agree a README can, and I noted composer does this, but the thought with the JSON was that in the README, it's easy(-ish) for a human to see, but sites like Packagist can't easily parse that out.
